special-pages/translations.mjs

```diff
@@ -0,0 +1,55 @@
+import { isLaunchFile } from '../scripts/script-utils.js';
+import { readdir } from 'fs/promises';
+import { join, basename } from 'node:path';
+import { readFileSync, writeFileSync } from 'node:fs';
+
+const paths = ['pages/new-tab'];
```

Comment:
This approach doesn't make as much sense in smaller special-pages, the NTP is a bit of special case because it includes many isolated widgets. So, for now I'm not concerned with having this as a hard-coded list of pages to process.